ls1intum/Artemis
# show team names when detecting plagiarism

### Checklist

### Motivation and Context


fixes #3509

### Steps for Testing


1. Check plagiarism on text and programming exercises (team and normal).
2. Check if team and student ids are shown.
